frontend/site/components/VideoCast.tsx

```diff
@@ -16,69 +15,45 @@ interface Prop {
   roomIds: string[]
   styles: CSSProperties[]
 }
-const SW_WSURL = "wss://webrtc.chofufes2022.ueckoken.club/"
+
 const SKYWAY_APIKEY =
   process.env.SKYWAY_APIKEY === undefined
-    ? "c07e8954-ce1b-4783-a45e-e8421ece83ce"
+    ? "2eb379e0-0374-4e3c-9674-d7415b0b7f27"
     : process.env.SKYWAY_APIKEY
 const SKYWAY_DEBUG_LEVEL = 2
 
 const VideoCast: FC<Prop> = ({ roomIds, styles }) => {
-  const webSocket = useRef<WebSocket>()
-  const [isWebSocketAvailable, setIsWebSocketAvailable] =
-    useState<boolean>(false)
-  const skywayPeer = useRef<Peer>()
   const [isPeerAvailable, setIsPeerAvailable] = useState<boolean>(false)
-
+  const skyWayPeer = useRef<Peer>()
   useEffect(() => {
-    webSocket.current = new WebSocket(SW_WSURL)
-    skywayPeer.current = new Peer({
+    skyWayPeer.current = new Peer({
       key: SKYWAY_APIKEY,
       debug: SKYWAY_DEBUG_LEVEL,
     })
-    webSocket.current.addEventListener("open", (event) => {
-      if (webSocket.current) {
-        setIsWebSocketAvailable(true)
-      }
-    })
-    webSocket.current.addEventListener("close", (event) => {
-      if (webSocket.current) {
-        setIsWebSocketAvailable(false)
-      }
-    })
-
-    skywayPeer.current.on("open", () => {
-      console.log("opened skyway peer")
+    skyWayPeer.current.on("open", (id) => {
+      console.log(`opened skyway peer id:${id}`)
       setIsPeerAvailable(true)
     })
-    skywayPeer.current.on("close", () => {
+    skyWayPeer.current.on("close", () => {
       console.log("closed skyway peer")
       setIsPeerAvailable(false)
     })
     return () => {
-      webSocket.current?.close()
-      skywayPeer.current?.destroy()
-      setIsWebSocketAvailable(false)
+      skyWayPeer.current?.destroy()
       setIsPeerAvailable(false)
     }
   }, [])
```

Comment:
```suggestion
  }, [setIsPeerAvailable])
```

たぶんこうやれば直りそう
（なんで去年は問題なかったのかは謎）

Reply:
その部分を修正しましたが、同じくエラーは出続けている感じです
運が良ければエラーが出ないような

Reply:
[setIsPeerAvailable, skyWayPeer]
のようにしたら正常に動きました

Reply:
@otariidae 色々と試しましたが、なかなかエラーが消えません。最初の一回目はうまくいきますが、それ以降が同じエラー「Error: Cannot connect to new Peer before connecting to SkyWay server or after disconnecting from the server.」がでて上手くいかないです。